### ISSUE TITLE
Print Movement

### ISSUE THREAD

**Issue by aetherical** [geordanr/xwing#504]
I really love this tool!  It's really, really helpful for creating squads and tweaking ideas.  Thank you very much for it.

I have one request -- in the GUI, when you hover over the ship, you can see the movements available on the dial. It'd be really helpful during play if this could be printed.

It  ranks on the order of "nice to have", but the tool is already really nice without it.

Thank you again!

**Comment by np-trivial** [comment]
I agree. This is about the only feature that Voidstate has that this tool doesn't.

**Comment by punkUser** [comment]
This! Was about to file a similar issue myself :)
